items/src/test/scala/weco/api/items/services/ItemUpdateServiceTest.scala

```diff
@@ -270,94 +301,167 @@ class ItemUpdateServiceTest
     val itemStates = Table(
       (
         "Sierra Responses",
+        "Content api Venue Responses",
         "Catalogue Work",
         "AccessCondition",
         "AvailableDates"
       ),
       (
         onHoldItemResponses(workWithAvailableItemNumber),
+        Seq(),
```

Comment:
Scala trivia: as well as `Seq()` or `List()` or `Seq.empty` or `List.empty` and so on, you can also just use `Nil` for any empty iterable